adapters/httpapi_winhttp.c

```diff
@@ -434,6 +434,9 @@ static HTTPAPI_RESULT InitiateWinhttpRequest(HTTP_HANDLE_DATA* handleData, HTTPA
     const wchar_t* requestTypeString;
     size_t requiredCharactersForRelativePath;
     wchar_t* relativePathTemp = NULL;
+    DWORD supported_protocols = WINHTTP_FLAG_SECURE_PROTOCOL_TLS1 | // TLS 1.0 is support for back-compat reasons (https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment)
```

Comment:
>supported_protocols [](start = 10, length = 19)

Nit: Other variables in this function follow the thisIsVariableName pattern.  "supportedProtocols' would be following convention instead of "_".

Reply:
good point. My bad. Confusing the guidelines. Will address it.

---
In reply to: [331714624](https://github.com/Azure/azure-c-shared-utility/pull/391#discussion_r331714624) [](ancestors = 331714624)